### PR TITLE
[MM-T2086] Cypress test for changing password and checking email and timestamp

### PR DESCRIPTION
**Pull request by Borknab**
#### Summary

Cypress test for MM-T2086:

Check that password setting _(in profile settings)_ shows password was updated at current time.

Additionally, check that an email notification about the password update is received.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/18818

### CODE REVIEW

**Review by josephbaylon — APPROVED** [review]
LGTM. Thanks @Borknab !!

**Comment by Borknab** [comment]
@josephbaylon @furqanmlk nice! Thanks guys, glad to contribute!

File: e2e/cypress/integration/account_settings/security/password_spec.js [hunk 1]
```diff
@@ -100,6 +109,37 @@ describe('Profile', () => {
...
+        cy.uiSave();
+
+        // # Create expected date and time formats
+        const date = now.format('MMM DD, YYYY');
+        const time = now.format('hh:mm A');
+
+        // # Create expected timestamp
+        const timestamp = `Last updated ${date} at ${time}`;
+
+        // # Verify that password description field contains valid timestamp
```
[the remark this answers is not in the document]
Comment (Borknab) [reply e2e/cypress/integration/account_settings/security/password_spec.js]: Hello! Thanks for the suggested change. I am new to this way of writing comments - what is the difference between leaving a # or a * ?

**Comment by Borknab** [comment]
Hello everyone! Why is the PR still not merged? Is there anything else I need to do?

**Comment by Borknab** [comment]
@furqanmlk @josephbaylon Hello guys! I received a notification that the PR got stale, though I don't understand why the PR has not been merged yet? Is there anything else I need to address?

File: e2e/cypress/integration/account_settings/security/password_spec.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (furqanmlk) [reply e2e/cypress/integration/account_settings/security/password_spec.js]: > Hello! Thanks for the suggested change. I am new to this way of writing comments - what is the difference between leaving a # or a * ?

Hi @Borknab apologize for responding late. If you look at the top of each spec file. 
there is description where to use `#` or `*`.
Here is the text 
// - [#] indicates a test step (e.g. # Go to a page)
// - [*] indicates an assertion (e.g. * Check the title)

[the remark this answers is not in the document]
Comment (Borknab) [reply e2e/cypress/integration/account_settings/security/password_spec.js]: Oh, I see, good to know! Thanks!

**Comment by josephbaylon** [comment]
@Borknab I've notified Furqan about this PR. As a side note, after you make the requested changes, please hit the re-request review to the person you wanted another review from so that he/she is notified. Otherwise, it may go off radar.

Thanks for notifying us directly though.

**Pull request merged**